src/torrent_clients/rtorrent.py

```
@@ -279,54 +267,48 @@ def rtorrent(self, path: str, torrent_path: str, torrent: Torrent, meta: Meta, l
         if local_path.lower() in path.lower() and local_path.lower() != remote_path.lower():
             path_dir = os.path.dirname(path)
             path = path.replace(local_path, remote_path)
-            path = path.replace(os.sep, '/')
+            path = path.replace(os.sep, "/")
             shutil.copy(fr_file, f"{path_dir}/fr.torrent")
             fr_file = f"{os.path.dirname(path)}/fr.torrent"
             modified_fr = True
-            if meta.debug:
-                logger.debug(f"[cyan]Modified fast resume file path because path mapping: {fr_file}")
+            logger.debug(f"[cyan]Modified fast resume file path because path mapping: {fr_file}")
         if meta.category in ("BOOK", "GAME") and len(filelist) > 1 and isdir or isdir is False:
```

Ruff (0.15.20)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)